src/OrchardCore.Modules/OrchardCore.Spatial/ViewModels/EditGeoPointFieldViewModel.cs

```diff
@@ -0,0 +1,22 @@
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using OrchardCore.ContentManagement;
+using OrchardCore.ContentManagement.Metadata.Models;
+using OrchardCore.Spatial.Fields;
+
+namespace OrchardCore.Spatial.ViewModels
+{
+    public class EditGeoPointFieldViewModel
+    {
+        public string Latitude { get; set; }
```

Comment:
Is that how we do for the numeric field (using string in the view model). Is the site's culture used to render and parse the value then?

Reply:
Yes